lib/controller/page/type/page.summary/page.summary.controller.js

```diff
@@ -62,17 +66,7 @@ SummaryController.setContents = (pageInstance, userData, res, hideChangeAction)
   return pageInstance
 }
 
-SummaryController.postValidation = async (pageInstance, userData) => {
-  const performsSubmit = checkSubmits(pageInstance, userData)
-  if (!performsSubmit) {
-    return pageInstance
-  }
-  const submissionId = uuidv4()
-  const submissionDate = (new Date()).getTime()
-  userData.setUserDataProperty('submissionId', submissionId)
-  userData.setUserDataProperty('submissionDate', submissionDate)
-  const userId = userData.getUserId()
-  const userToken = userData.getUserToken()
+SummaryController.attachEmailSubmissions = async (submissionId, userData, submissions) => {
```

Comment:
I would move this and anything to do with the submission into a separate submission module. It was sheer laziness that has conflated the summary code.

Reply:
sure, i think that's a refactor for later